js/renovation/ui/responsive_box/responsive_box.tsx

```diff
@@ -5,8 +5,31 @@ import { Widget } from '../common/widget';
 import { ResponsiveBoxProps } from './responsive_box_props';
 import { combineClasses } from '../../utils/combine_classes';
 
-export const viewFunction = (): JSX.Element => {
-  const cssClasses = combineClasses({ 'dx-responsivebox': true });
+import { hasWindow } from '../../../core/utils/window';
+import domAdapter from '../../../core/dom_adapter';
+import { ScreenSizeQualifier } from './types';
+import { convertToScreenSizeQualifier } from './screen_utils';
+
+const HD_SCREEN_WIDTH = 1920;
+const RESPONSIVE_BOX_CLASS = 'dx-responsivebox';
+const SCREEN_SIZE_CLASS_PREFIX = `${RESPONSIVE_BOX_CLASS}-screen-`;
+
+export const viewFunction = (viewModel: ResponsiveBox): JSX.Element => {
+  const getCurrentScreenSizeQualifier = (): ScreenSizeQualifier => {
+    const screenWidth = hasWindow()
+      ? domAdapter.getDocumentElement().clientWidth
```

Comment:
There was "return hasWindow() ? $(window).width() : HD_SCREEN_WIDTH;"
But "domAdapter.getDocumentElement().clientWidth" is used now.
Why?

https://js.devexpress.com/Documentation/21_1/ApiReference/UI_Components/dxResponsiveBox/Configuration/#screenByWidth
"accepts the screen's width" - this description seems to be outdated? There is no "screen's width" anymore.

Reply:
> But "domAdapter.getDocumentElement().clientWidth" is used now. Why

It is analog of the computing of the window width except scrollbar width.
Inner width and outerWidth of the window object returns width without checking of the visibility of the scrollbar
But clientWidth of the document check it:

![image](https://user-images.githubusercontent.com/57402891/117014394-6c600b80-acf9-11eb-954a-9eacc3226660.png)